dask/dataframe/io/tests/test_csv.py

```diff
@@ -34,6 +35,9 @@
 # List of available compression format for test_read_csv_compression
 compression_fmts = [fmt for fmt in compr] + [None]
 
+CONVERT_STRING = config.get("dataframe.convert_string")
+OBJECT_DTYPE = pd.StringDtype("pyarrow") if CONVERT_STRING else object
```

Comment:
Let's move these to a utils file instead of defining in a bunch of different places. It's probably also worth making these callables so they'll always be valid. Currently, for example, `OBJECT_DTYPE` is defined when this module is first run, but later we might modify the `dataframe.convert_string` config option and `OBJECT_DTYPE` don't be accurate anymore. Maybe something like

```python
# dask/dataframe/utils.py

def get_string_dtype():
    return pd.StringDtype("pyarrow") if dask.config.get("dataframe.convert_string") else object

```

Reply:
Moved to `dask.dataframe.utils`.